src/util/symbols.js

```diff
@@ -1,10 +1,11 @@
-export const connected = '____skate_connected';
-export const created = '____skate_created';
+import createSymbol from './create-symbol';
+export const connected = createSymbol();
+export const created = createSymbol();
 export const name = '____skate_name';
-export const props = '____skate_props';
-export const ref = '____skate_ref';
-export const renderer = '____skate_renderer';
-export const rendering = '____skate_rendering';
-export const rendererDebounced = '____skate_rendererDebounced';
-export const shadowRoot = '____skate_shadowRoot';
-export const updated = '____skate_updated';
+export const props = createSymbol();
```

Comment:
I think this will break cross-version compatibility, because our idom integration looks at the `elem[$props]` to determine if it should set as attribute as a property doesn't it?